packages/orbit-components/src/Select/README.md

```diff
@@ -88,3 +89,7 @@ class Component extends React.PureComponent<Props> {
   }
 }
 ```
+
+## Accessibility
+
+- The `ariaLabel` prop allows you to specify an `aria-label` attribute for the popover content component. This attribute provides additional information to screen readers, enhancing the accessibility of the component. By using `ariaLabel`, you can ensure that users who rely on assistive technologies receive the necessary context and information about the component's purpose and functionality.
```

Comment:
Popover? 😅 

I feel we can be more specific here. In this case, the `aria-label` works for when we don't want a visible written `label` (because we assume the context of the select is given visually somehow) but we need to label it for screen readers.
TL;DR: The `aria-label` **must** be used when `label` is not defined, for whatever reason